templates/secret.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sc-secret
+  annotations:
+    tekton.dev/docker-0: https://docker.io
+type: kubernetes.io/basic-auth
+stringData:
+  src_username: <>
+  src_password: <>
+  dest_username: <>
+  dest_password: <>
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sc-account
+secrets:
+  - name: sc-secret
```

Comment:
Here we should mention the service account needs to be edited (`kubectl edit serviceaccount ...`), you can show a example on `README.md`.